requirements/ingest-hubspot.in

```diff
@@ -0,0 +1,2 @@
+hubspot-api-client
+urllib3<=1.26.17
```

Comment:
`urllib3` already gets pulled in from the base dependency. Usually we start this file off with 
```
-c constraints.in
-c base.txt
```
Which will pull those in.